src/rust/Cargo.toml

```diff
@@ -3,13 +3,24 @@ members = ["aggregator", "api", "db", "types"]
 exclude = ["dependencies", "sdk"]
 
 [workspace.dependencies]
-bigdecimal = "^0.3.0"
-chrono = "^0.4.24"
-dotenvy = "^0.15.6"
-envy = "^0.4.2"
-futures-util = "^0.3.26"
-serde = { version = "^1.0.149", features = ["derive"] }
-serde_json = "^1.0.93"
-sqlx = { version = "^0.6.2", features = ["runtime-tokio-rustls"] }
-thiserror = "^1.0.38"
-tokio = "^1.25.0"
+axum = "0.6.19"
```

Comment:
Why do dependencies that are only used in one workspace member need to be in the workspace root?

Reply:
Even if you put a dependency only in 1 crate, its beneficial to have it at the workspace level because the entire workspace shares the same Cargo.lock file, so by doing this it stops someone in the future placing a conflicting version dependency in a member of the workspace.

Reply:
Hmmmm sure